### ISSUE TITLE
10.0.17763.4644 not supported

### ISSUE THREAD

**Issue by r2x-ch** [stascorp/rdpwrap#2274]
Version 10.0.17763.4644 is not supported

![sshot-17](https://github.com/stascorp/rdpwrap/assets/33598321/4e00b4fa-73c0-42c5-ae86-5a58c665944b)

File termsrv.zip includes termsrv.dll

Thank you!!

[termsrv.zip](https://github.com/stascorp/rdpwrap/files/12021280/termsrv.zip)

**Comment by Glavonja84** [comment]
Hi,
Would need that offset as well. BR

**Comment by gimadeevmr** [comment]
+1

**Comment by sashaqwert** [comment]
https://github.com/sebaxakerhtc/rdpwrap.ini/commit/a04cab4f99b83234f7de58a35237c15deac95264

**Comment by houchangjun1996** [comment]
[10.0.17763.4644]
LocalOnlyPatch.x64=1
LocalOnlyOffset.x64=8FF11
LocalOnlyCode.x64=jmpshort
SingleUserPatch.x64=1
SingleUserOffset.x64=0D579
SingleUserCode.x64=Zero
DefPolicyPatch.x64=1
DefPolicyOffset.x64=206A5
DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
SLInitHook.x64=1
SLInitOffset.x64=3180C
SLInitFunc.x64=New_CSLQuery_Initialize

[10.0.17763.4644-SLInit]
bInitialized.x64      =11E278
bServerSku.x64        =11E27C
lMaxUserSessions.x64  =11E280
bAppServerAllowed.x64 =11E288
bRemoteConnAllowed.x64=11E28C
bMultimonAllowed.x64  =11E290
ulMaxDebugSessions.x64=11E294
bFUSEnabled.x64       =11E298

**Comment by JDesign23** [comment]
![image](https://github.com/stascorp/rdpwrap/assets/139445327/656a62d2-438c-4401-bf31-9a50f077c3d0)
Winver: 1809.17763.4645 (Windows 10 LTSC) not working after adding [10.0.17763.4644] offsets only showing that supporting, but not working. Only one session at a time allowed.

**Comment by r2x-ch** [comment]
> ![image](https://user-images.githubusercontent.com/139445327/253227355-656a62d2-438c-4401-bf31-9a50f077c3d0.png) Winver: 1809.17763.4645 (Windows 10 LTSC) not working after adding [10.0.17763.4644] offsets only showing that supporting, but not working. Only one session at a time allowed.

Have you tried the solution from [houchangjun1996](https://github.com/houchangjun1996)? Worked for me on Server 2019.

**Comment by JDesign23** [comment]
> > ![image](https://user-images.githubusercontent.com/139445327/253227355-656a62d2-438c-4401-bf31-9a50f077c3d0.png) Winver: 1809.17763.4645 (Windows 10 LTSC) not working after adding [10.0.17763.4644] offsets only showing that supporting, but not working. Only one session at a time allowed.
> 
> Have you tried the solution from [houchangjun1996](https://github.com/houchangjun1996)? Worked for me on Server 2019.

Yes, my attached screenshot made after updating rdpwrap.ini but it`s not working at all :( only good diagnostics. Reinstalling, rebooting, restarting services not made any different.

**Comment by r2x-ch** [comment]
> > > ![image](https://user-images.githubusercontent.com/139445327/253227355-656a62d2-438c-4401-bf31-9a50f077c3d0.png) Winver: 1809.17763.4645 (Windows 10 LTSC) not working after adding [10.0.17763.4644] offsets only showing that supporting, but not working. Only one session at a time allowed.
> > 
> > 
> > Have you tried the solution from [houchangjun1996](https://github.com/houchangjun1996)? Worked for me on Server 2019.
> 
> Yes, my attached screenshot made after updating rdpwrap.ini but it`s not working at all :( only good diagnostics. Reinstalling, rebooting, restarting services not made any different.

Okay, the only thing that comes to my mind: Check if you have an empty line after the code in the ini file, otherwise it won't work.

**Comment by Glavonja84** [comment]
Hi,
Had problems myself, as the service and listener were not working (had 2 simultaneous connects only). 
The problem was I updated rdpwrap.ini, but once I replaced the whole file and did not just update it, it worked after restarting it.
Had the problem that it was fully supported, but it was not working.

https://raw.githubusercontent.com/sebaxakerhtc/rdpwrap.ini/a04cab4f99b83234f7de58a35237c15deac95264/rdpwrap.ini
BR

**Comment by JDesign23** [comment]
> > > > ![image](https://user-images.githubusercontent.com/139445327/253227355-656a62d2-438c-4401-bf31-9a50f077c3d0.png) Winver: 1809.17763.4645 (Windows 10 LTSC) not working after adding [10.0.17763.4644] offsets only showing that supporting, but not working. Only one session at a time allowed.
> > > 
> > > 
> > > Have you tried the solution from [houchangjun1996](https://github.com/houchangjun1996)? Worked for me on Server 2019.
> > 
> > 
> > Yes, my attached screenshot made after updating rdpwrap.ini but it`s not working at all :( only good diagnostics. Reinstalling, rebooting, restarting services not made any different.
> 
> Okay, the only thing that comes to my mind: Check if you have an empty line after the code in the ini file, otherwise it won't work.

Yes, there was an empty line, thank you very much. All is working great now.
